opennms-dao-api/src/main/java/org/opennms/netmgt/dao/api/ResourceStorageDao.java

```diff
@@ -117,17 +117,23 @@
 
     /**
      * Sets the specified (resource-level) attribute at the given path.
+     * Deprecated: if needed store collector specific string attributes in a KV store
      */
+    @Deprecated
```

Comment:
This is the only method we need to deprecate - strings should only be set and persisting metrics - but we can still use this interface to retrieve their values.